Notes/7-Scope-LexEnv.md

```diff
@@ -43,6 +43,7 @@ function a() {
 
 - This is the intuition behind **scope**
 - Scope is directly dependent on the lexical environment
+- "Scope" means where i can accessible a specific variable or function in the code
```

Comment:
It should be 
`Scope is where I can access a specific variable or function in the code`
Kindly change it :)